.github/workflows/build.yml

```diff
@@ -12,6 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
 
+    strategy:
+      matrix:
+        platform: [ jvm, js, linuxX64, macosX64, mingwX64 ]
```

Comment:
We cannot build `macosX64` or `mingwX64` on ubuntu? 🤔

Reply:
Done, please @nomisRev , check whenever you have time